docs/04-operation-guides/operations/10-backup-kyma.md

```diff
@@ -1,206 +0,0 @@
----
-title: Back up Kyma
```

Comment:
Is that document covered somewhere?

Reply:
We have that general document -> https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-environment-backup?version=Cloud. Let's enrich it with the details we want to remove. I'm leaving the doc for the time being. [Issue](https://github.tools.sap/kyma/backlog/issues/4813) created.